### DISCUSSION TITLE
DS18B20 recognized as HIH

### DISCUSSION THREAD

**Discussion by ErikApption** [arendst/Tasmota#22381] [Support request]
Any suggestions to prevent my DS18B20 to avoid being recognized as HIH in Tasmota and possibly force recognition? I am on a ESP32 and I followed the exact wiring guides from the Tasmota guide. Issue has been intermittent from v12 to latest release v14

**Comment by sfromis** [comment]
If by "HIH" mean the "Honeywell HIH series temperature and humidity sensor", that is an I2C device, and should not in any way be detected as a DS18B20. Or what specifically do you mean by "HIH", if not that. What do you see when it is wrongly recognized?

**Reply by ErikApption** [reply]
> If by "HIH" mean the "Honeywell HIH series temperature and humidity sensor", that is an I2C device, and should not in any way be detected as a DS18B20. Or what specifically do you mean by "HIH", if not that. What do you see when it is wrongly recognized?

Might be multiple issues. I think I had I2C setup without anything connected and Tasmota found a non-existent HIH. My DS18B20 is not recognized unfortunately.

**Comment by ErikApption** [comment]
The HIH is without any doubt a mystery however the DS18x20 is not... The sensor was physically unplugged lol

**Reply by sfromis** [reply]
Defining pins without them being wired to anything is not at all good practice.

Still, disabling irrelevant I2C drivers may make sense if they cause "surprises"....
https://tasmota.github.io/docs/I2CDEVICES/
